examples/systemcalls/systemcalls.c

```diff
@@ -93,7 +105,26 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+    const int outputFd = open(outputfile, O_WRONLY | O_CREAT | O_TRUNC, S_IRWXU | S_IRWXG | S_IRWXO);
+    fflush(stdout);
+    const pid_t pid = fork();
```

Comment:
@bennowotny think about how you could avoid duplicating logic between these two functions.